doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
+\begin{table}[htb]
+	\center
+	\begin{tabular}{|c|cccccc|}
+		\hline
+		Global & & Number of Processors & & \\
+		Refinements & 4 & 8 & 12 & 16
+		\\ \hline
+		5 & 28.1 seconds & 19.8 seconds & 19.6 seconds & 17.1 seconds \\
+		6 & 3.07 minutes & 1.95 minutes & 1.49 minutes & 1.21 minutes \\
+		7 & 23.33 minutes & 13.92 minutes & 9.87 minutes & 7.33 minutes \\
+		8 & 3.08 hours & 1.83 hours & 1.30 hours & 56.33 minutes \\
+		9 & 1.03 days & 15.39 hours & 10.44 hours & 7.53 hours \\
+		10 & More than 6 days & More than 6 days & 3.39 days & 2.56 days \\ \hline
+	\end{tabular}
+	\caption{\it Average runtimes for the Van Keken Benchmark with smoothed initial conditions.  These times are for the entire computation, a final time step of 2000.  All of these tests were run using ASPECT version 1.3 in release mode, and used different numbers of processors on one node on the BlueRidge cluster of ARC at Virginia Tech.}
+	\label{tab:runtime-table}
+\end{table}
+
+This convergence is due to changing the smoothing parameter, which controls how much of the problem is smoothed over.  As the parameter is increased. the smoothed boundary grows and vice versa.  As the smoothed boundary shrinks it becomes sharper until the original discontinuous behavior is revealed.  As it grows, the two layers eventually become one large, transitioning layer rather than two distinct layers separated by a boundary.  These effects can be seen in Fig.~\ref{fig:vk-5}.  The overall effect is that the secondary rise is at different times based on these conditions.  In general, as the smoothing parameter is decreased, the smoothed boundary shrinks and the plumes rise more quickly.  As it is increased, the boundary grows and the plumes rise more slowly.  This trend can be used to force a more accurate convergence from the secondary plumes.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.8\textwidth]{cookbooks/benchmarks/van-keken/smoothing-parameter.png}
+	\caption{\it Van Keken Benchmark using smoothed out interface: compositional field at time t=0 using smoothing parameter size: a) 0.0039, b) 0.0078, c) 0.0156, d) 0.0234, e) 0.0312, f) 0.0390, g) 0.0468, h) 0.0546, i) 0.0624.}
```

Comment:
Enclose `t=0` in dollar signs to have it rendered as a formula.

Can you add in the caption which mesh you used for this computation?